### DISCUSSION TITLE
Operation not permitted when modifying files

### DISCUSSION THREAD

**Discussion by atnasss** [containerd/nerdctl#2012] [Q&A]
I run into a problem when copying files during image build and then trying to access these files at runtime. Here the steps I used to verify the problem (verified with Vagrant VM ubuntu/jammy64 and downloaded nerdctl full: nerdctl-full-1.2.0-linux-amd64.tar.gz):
I run a basic image, copying 1 file:

Dockerfile:
```
FROM ubuntu:jammy
COPY my_project/manage.py /
```
I build and run the image passing a command : sh -c "sleep 3600"
I connect to the running container:
```
nerdctl exec -ti [container] bash
```
and try to modify the copied file. It works.
I create another container with the same image but this time I connect specifying a user:
```
nerdctl exec -ti -u root [container] bash
```
If I try to modify the copied file (manage.py) I receive an error:
```
echo "aaaaa" > manage.py
bash: manage.py: Operation not permitted
```
And from that moment even reconnecting without the -u argument does not allow me to modify the file. If I stop/start the container it works again.
Do you know what I'm doing wrong? Or is this supposed to be the expected behavior?

**Comment by fahedouch** [comment]
are you using rootless mode ?

**Reply by atnasss** [reply]
No, rootful
